### PR TITLE
Homework3: POSTGRESQL AND LAYERED ARCHITECTURE

### PR DESCRIPTION
**Pull request by JieLiuRiver**

### CODE REVIEW

**Review by JonCa1 — COMMENTED** [review]
test

File: src/mapper/userMapper.ts [hunk 1]
```diff
@@ -0,0 +1,8 @@
+
```
Comment (JonCa1) [inline src/mapper/userMapper.ts]: put helper methods in a helpers/utils folder

File: src/config/index.ts [hunk 2]
```diff
@@ -0,0 +1,9 @@
+export default {
```
Comment (JonCa1) [inline src/config/index.ts:1]: maintain db related config in .env/.env.local file for different env

File: src/router/controller/index.ts [hunk 3]
```diff
@@ -0,0 +1,5 @@
+import UserController from './userController';
```
Comment (JonCa1) [inline src/router/controller/index.ts]: just leave controllers folder on the root path on src is fine

File: src/mapper/userMapper.ts (hunk 1 above)
Comment (JieLiuRiver) [reply src/mapper/userMapper.ts]: Good suggestion

File: src/router/controller/index.ts (hunk 3 above)
Comment (JieLiuRiver) [reply src/router/controller/index.ts]: Ok, Let me adjust it

File: src/config/index.ts (hunk 2 above)
Comment (JieLiuRiver) [reply src/config/index.ts:1]: The `env` homework will be involved in the later course, so I won't adjust it in this branch to avoid conflict

**Pull request merged**